HLTrigger/Configuration/python/deepTauAtHLT.py

```diff
@@ -0,0 +1,357 @@
+    process.HLTHPSMediumChargedIsoPFTauSequenceReg.insert(-1, process.chargedIsoPtSumdR03)
+    process.HLTHPSMediumChargedIsoPFTauSequenceReg.insert(-1, process.neutralIsoPtSum)
+    process.HLTHPSMediumChargedIsoPFTauSequenceReg.insert(-1, process.neutralIsoPtSumdR03)
+    process.HLTHPSMediumChargedIsoPFTauSequenceReg.insert(-1, process.puCorrPtSum)
+    process.HLTHPSMediumChargedIsoPFTauSequenceReg.insert(-1, process.footprintCorrection)
+    process.HLTHPSMediumChargedIsoPFTauSequenceReg.insert(-1, process.footprintCorrectiondR03)
+    process.HLTHPSMediumChargedIsoPFTauSequenceReg.insert(-1, process.neutralIsoPtSumWeight)
+    process.HLTHPSMediumChargedIsoPFTauSequenceReg.insert(-1, process.neutralIsoPtSumWeightdR03)
+    process.HLTHPSMediumChargedIsoPFTauSequenceReg.insert(-1, process.photonPtSumOutsideSignalCone)
+    process.HLTHPSMediumChargedIsoPFTauSequenceReg.insert(-1, process.photonPtSumOutsideSignalConedR03)
+    process.HLTHPSMediumChargedIsoPFTauSequenceReg.insert(-1, process.hltFixedGridRhoFastjetAll)
+
+
+    file_names = [
+    				'core:RecoTauTag/TrainingFiles/data/DeepTauId/deepTau_2017v2p6_e6_core.pb',
+    				'inner:RecoTauTag/TrainingFiles/data/DeepTauId/deepTau_2017v2p6_e6_inner.pb',
+    				'outer:RecoTauTag/TrainingFiles/data/DeepTauId/deepTau_2017v2p6_e6_outer.pb',
+    			]
+
+    wp_names = ["0.", "0."]
+
+    process.deepTauProducer = cms.EDProducer("DeepTauId",
+    taus                            = cms.InputTag("hltHpsPFTauProducerReg"),
+    pfcands                         = cms.InputTag('hltParticleFlowReg'),
+    vertices                        = cms.InputTag('hltPixelVertices'),
+    rho                             = cms.InputTag('hltFixedGridRhoFastjetAll'),
+    graph_file                      = cms.vstring(file_names),
+    mem_mapped                      = cms.bool(False),
+    version                         = cms.uint32(2),
+    debug_level                     = cms.int32(0),
+    disable_dxy_pca                 = cms.bool(True),
+    is_online                 		  = cms.bool(True),
+    pfTauTransverseImpactParameters = cms.InputTag('hpsPFTauTransverseImpactParameters'),
+    chargedIsoPtSum                 = cms.InputTag('chargedIsoPtSum'),
+    chargedIsoPtSumdR03             = cms.InputTag('chargedIsoPtSumdR03'),
+    neutralIsoPtSum                 = cms.InputTag('neutralIsoPtSum'),
+    neutralIsoPtSumdR03             = cms.InputTag('neutralIsoPtSumdR03'),
+    puCorrPtSum                     = cms.InputTag('puCorrPtSum'),
+    footprintCorrection             = cms.InputTag('footprintCorrection'),
+    neutralIsoPtSumWeight           = cms.InputTag('neutralIsoPtSumWeight'),
+    neutralIsoPtSumWeightdR03       = cms.InputTag('neutralIsoPtSumWeightdR03'),
+    photonPtSumOutsideSignalCone    = cms.InputTag('photonPtSumOutsideSignalCone'),
+    photonPtSumOutsideSignalConedR03 = cms.InputTag('photonPtSumOutsideSignalConedR03'),
```

Comment:
Could you please run `scram build code-checks` and `scram build code-format`? I fear the latter will kill this almost perfect alignment here, but then we don't have to talk about fixing it. The checks and format applied by these scripts are anyway a requirement to pass the integration tests.